setup.py

```diff
@@ -83,7 +83,7 @@ def get_requirements(requirements_file):
 
 README = open(os.path.join(base_path, "README.rst")).read()
 CHANGELOG = open(os.path.join(base_path, "CHANGELOG.rst")).read()
-REQUIREMENTS, DEPENDENCY_LINKS = get_requirements(os.path.join(base_path, 'requirements', 'base.txt'))
+REQUIREMENTS, DEPENDENCY_LINKS = get_requirements(os.path.join(base_path, 'requirements', 'base.in'))
```

Comment:
Should this be test-master.txt?

Reply:
no; the idea is that when this library declares its dependencies, we err towards not being opinionated about the versions, so we use base.in because that has all the dependencies listed but only specifies versions for packages we need to do so for. test-master.txt is for getting specific versions to run tests against, ideally the ones that match what's in edx-platform.

Reply:
i see i see. thanks